ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetBulk.java

```diff
@@ -116,88 +134,96 @@ public CliCommand init(final Arguments args) throws Exception {
 
     @Override
     public DefaultResult call() throws Exception {
-        final Ds3ClientHelpers.ObjectChannelBuilder getter
-                = this.discard ?  new MemoryObjectChannelBuilder(DEFAULT_BUFFER_SIZE, DEFAULT_FILE_SIZE)
-                : new FileObjectGetter(this.outputPath);
+        final Ds3ClientHelpers.ObjectChannelBuilder getter;
+        if (this.pipe) {
+            getter = new PipedFileObjectGetter(this.outputPath, FileUtils.normalizedObjectNames(this.pipedFileNames));
```

Comment:
I'm a bit confused why we need a new FileObjectGetter.  What does FileObjectGetter not do that the new PipedFileObjectGetter do?